src/sentry_plugins/jira_ac/views.py

```diff
@@ -100,6 +102,8 @@ def get_context(self) -> MutableMapping[str, str]:
 
     @transaction_start("JiraConfigView.get")
     def get(self, request: Request, *args: Any, **kwargs: Any) -> HttpResponse:
+        if not features.has("organizations:jira-ac-plugin", request.user.get_orgs()):
```

Comment:
I don't think you can pass `has()` a list of orgs. You might need to use `any_organization_has_feature()`.

Reply:
I had a vivid memory you wrote something to address this but I thought it was just rolled into `has()` 🙃